packages/kernel-hardening-checker/default.nix

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
```

Comment:
Please update license year to 2024 instead of 2023
"# Copyright 2022-202**4** TII (SSRC) and the Ghaf contributors"